docker/docker-compose.centos-6.111.yaml

```diff
@@ -6,7 +6,7 @@ services:
     image: netty:centos-6-1.11
     build:
       args:
-        java_version : "11.0.25-zulu"
+        java_version : "17.0.14-zulu"
```

Comment:
this is not what we want... our docker compose config should use the correct version for it. We use different configs for different jdk versions

Reply:
The `.111` in the file name refers to this compose file being for Java 11 (JDK 1.11).

This change is also irrelevant now, because in https://github.com/netty/netty/pull/14910 we bump the minimum Java version for the `main` branch to Java 22.